alembic/versions/165f21d7f610_create_new_garden_table.py

```diff
@@ -0,0 +1,27 @@
+"""Create New Garden Table
```

Comment:
Having a proper many to many relationship really made this less brain-bendy, AND if we we were to upscale the app that part of functionality is already built in now.  Glad we spent the time on this instead of trying to do it in some weird kinda way.